src/webview/js/js.js

```diff
@@ -40,31 +42,17 @@ import type { MessageListEvent } from '../webViewEventHandlers';
 /** Like RN's `Platform.OS`. */
 const platformOS = window.navigator.userAgent.match(/iPhone|iPad|iPod/) ? 'ios' : 'android';
 
-/**
- * Convert an array-like object to an actual array.
- *
- * A substitute for `Array.from`, which doesn't exist in the WebViews of some
- * of our supported platforms.
- */
-function arrayFrom<T>(arrayLike: HTMLCollection<T>): T[] {
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects
-  return Array.prototype.slice.call(arrayLike);
-}
+/* eslint-disable no-extend-native */
 
-/**
- * A substitute for `Array.from`, for `NodeList` input.
- *
- * See `arrayFrom`.  Duplicated to satisfy Flow.
- */
-// On at least Flow v0.78.0, writing `HTMLCollection<T> | NodeList<T>`
-// worked fine for `HTMLCollection` arguments, but not `NodeList`.  Shrug.
-function arrayFromNodes<T>(arrayLike: NodeList<T>): T[] {
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects
-  return Array.prototype.slice.call(arrayLike);
+/* Polyfill Array.from. Native in Chrome 45 and at least Safari 13.
+   Leaves out some of the fancy features (see MDN). */
+if (!Array.from) {
+  // $FlowFixMe (polyfill)
+  Array.from = function from(arr: $ArrayLike<T>): Array<T> {
```

Comment:
I'm afraid that what I used was [this](https://www.saltycrane.com/flow-type-cheat-sheet/latest/), which I obtained by typing "flow arraylike" into Google and crossing my fingers.

... and as it turns out, it doesn't actually exist until Flow 0.104.0. (We're at 0.92.1.) Flow is only declining to complain about it because the `// $FlowFixMe (polyfill)` comment swallows *all* the errors.

I've added a definition as `ArrayLike`, copied from the source (it's [very short](https://github.com/facebook/flow/blob/v0.104.0/lib/core.js#L308)) – but it's theater, rather than a functional definition. :disappointed:

Reply:
> I'm afraid that what I used was [this](https://www.saltycrane.com/flow-type-cheat-sheet/latest/), which I obtained by typing "flow arraylike" into Google and crossing my fingers.

Ooh, handy! That seems no worse than a lot of the information one has to rely on in working with Flow :grimacing: .

The links to the implementation seem handy too.

Reply:
Added a quick link to that cheat sheet, along with some other new text on finding Flow documentation (substitutes), here: ec8d9157d